docker/indexer/stages/processing/processing.go

```diff
@@ -55,6 +69,9 @@ type Stage struct {
 	PubSubOutstandingMessages int
 }
 
+// bucketCount should be a divisor of 2^16
```

Comment:
maybe also add a note that changing this will require some backfilling/reindexing?